plugin/trino-google-sheets/src/test/java/io/trino/plugin/google/sheets/SheetsQueryRunner.java

```diff
@@ -35,22 +36,23 @@ private SheetsQueryRunner() {}
 
     public static DistributedQueryRunner createSheetsQueryRunner(
             Map<String, String> extraProperties,
-            Map<String, String> connectorProperties)
+            Map<String, String> connectorProperties,
+            Optional<String> metadataSheetId)
             throws Exception
     {
         DistributedQueryRunner queryRunner = DistributedQueryRunner.builder(createSession())
                 .setExtraProperties(extraProperties)
                 .build();
         try {
             // note: additional copy via ImmutableList so that if fails on nulls
-            connectorProperties = new HashMap<>(ImmutableMap.copyOf(connectorProperties));
-            connectorProperties.putIfAbsent("gsheets.credentials-path", getTestCredentialsPath());
-            connectorProperties.putIfAbsent("gsheets.metadata-sheet-id", TEST_METADATA_SHEET_ID);
-            connectorProperties.putIfAbsent("gsheets.max-data-cache-size", "1000");
-            connectorProperties.putIfAbsent("gsheets.data-cache-ttl", "5m");
+            final Map<String, String> properties = new HashMap<>(ImmutableMap.copyOf(connectorProperties));
+            properties.putIfAbsent("gsheets.credentials-path", getTestCredentialsPath());
+            metadataSheetId.ifPresent(sheetId -> properties.putIfAbsent("gsheets.metadata-sheet-id", sheetId));
+            properties.putIfAbsent("gsheets.max-data-cache-size", "1000");
+            properties.putIfAbsent("gsheets.data-cache-ttl", "5m");
 
             queryRunner.installPlugin(new SheetsPlugin());
-            queryRunner.createCatalog(GOOGLE_SHEETS, GOOGLE_SHEETS, connectorProperties);
+            queryRunner.createCatalog(GOOGLE_SHEETS, GOOGLE_SHEETS, properties);
```

Comment:
Please revert the rename, or extract it as another commit.

Reply:
reverted the rename.